### PR TITLE
Site Template: write a Gemfile which is educational to the new site

### LINKED ISSUE #4539: Add version to _config.yml and warn if a different version is used on build
If we could add a version field to _config.yml this would save debugging time when using multiple versions of Jekyll. An example _config.yml would be:

```
name: Version Test
author: george@cloudcannon.com
version: 3.0.3
```

If you ran `jekyll serve` and it ran anything other than 3.0.3 it would output:

```bash
$ jekyll serve
Running Jekyll XXX but site expected 3.0.3 see http://jekyllrb.com/docs/upgrading/2-to-3/ for potential changes
```

Running the correct version would print nothing

```bash
$ jekyll _3.0.3_ serve
...
```

This matches a feature on CloudCannon at http://docs.cloudcannon.com/building/jekyll/.

### PR DESCRIPTION
**Pull request by parkr**
Fixes #4539. Kind of.

- [x] Create Gemfile template
- [x] Test it

### CODE REVIEW

File: lib/jekyll/commands/new.rb [hunk 1]
```diff
@@ -59,6 +63,27 @@ def initialized_post_name
         end
 
         private
+        
+        def gemfile_contents
+          <<-RUBY
+source 'https://rubygems.org'
+  
+# Hello! This is where you manage which Jekyll version is used to run.
+# When you wwant to use a different version, change it below, save the
```
Comment (benbalter) [inline lib/jekyll/commands/new.rb]: `wwant` -> `want`

**Comment by benbalter** [comment]
I'm a big fan of this. Was kinda surprised the default site didn't come with a Gemfile. Think it'd go a long way to help ease non-Ruby users into the Jekyll world, and if a user doesn't use Bundler, no harm done.

**Comment by parkr** [comment]
@jekyll/stability, look good to you?

**Comment by parkr** [comment]
@jekyllbot: merge +minor

**Pull request merged**